askama_shared/src/generator.rs

```diff
@@ -208,15 +208,15 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
     // Implement Actix-web's `Responder`.
```

Comment:
Not sure what you mean by "`Future`s not being found"?

Reply:
The previous version of actix-web used `type Future` while this does just return a `HttpResponse` without any Result or future. the hacky fix is to use expect instead of returning the error. As I think we should return the error but I didnt find any hint in the actix docs on how to return an error from the respond_to method :/ I am probably missing something I think but also didnt research hard enough.

Also enjoy your holiday :)

Reply:
I'm not sure how to make progress on this. I definitely don't want to call `expect()` here. Maybe you can ask the Actix folks how we're supposed to return an error in this case?